### PR TITLE
Adding the project list

### PR DESCRIPTION
**Pull request by craiglpeters**
Please review this way of communicating about our projects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+|  | [AKS Engine](https://github.com/Azure/aks-engine) <br> ([releases](https://github.com/Azure/aks-engine/releases)) | Self-managed clusters on Azure | Azure: incubating <br> API: N/A | [#aks-engine-users](https://kubernetes.slack.com/archives/CU3N85WJK) <br> [GitHub issues](https://github.com/Azure/aks-engine/issues) | [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/) <br> [Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview) |
+|  | [Cluster API Azure Provider](https://sigs.k8s.io/cluster-api-provider-azure) <br> ([releases](https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases)) <br> [Tests](https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-azure) | Self-managed clusters on Azure using Cluster API | CNCF: incubating <br> API: v1alpha2 | [#cluster-api-azure](https://kubernetes.slack.com/archives/CEX9HENG7) <br> [kubernetes-sig-cluster-lifecycle@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) <br> [GitHub issues](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues) |  |
```
Comment (CecileRobertMichon) [inline README.md]: API: v1alpha3 now :)

File: README.md [hunk 2]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+
+# Azure Container Compute Upstream Projects
+
+This list of projects is maintained by the Azure container compute upstream team (TODO: how to contact the team). This list is inteded to help you make informed decisions about what projects to use (or not use) in the context of your goals (e.g. proof of concept vs. production). To make this decision you need to consider your goals, your need for formal support, the project's [maturity](#Maturity), governance, version level, and your willingness to work in open source.
+
+## Support
+
+Projects listed on this page are open source that Microsoft maintain or contribute to. These projects are [**NOT** covered by the Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). To get help please search open issues using the links. If your issue isn't already represented, please open a new one. Consider that if you consume one of these projects as a part of a Microsoft or Azure product or service, you may be eligible for [support under that product](https://support.microsoft.com/en-us/hub/4343728/support-for-business).
```
Comment (jackfrancis) [inline README.md]: The "Consider that..." sentence (the last one) seems to contradict the one about the projects not being Azure-supported.

Comment (craiglpeters) [reply README.md]: I'm trying to leave room for the product or service to support. I'm open to suggestions

Comment (jackfrancis) [reply README.md]: how about "you may be eligible for additional business support in lieu of project-specific support".

File: README.md [hunk 3]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+
+## Project list
+
+| Project Area | Project & (artifacts) | Goal | Project State & <br> API Version | Communication | Use on Azure |
+|---|---|---|---|---|---|
+| Kubernetes Cluster Management |  |  |  |  |
+|  | [AKS Engine](https://github.com/Azure/aks-engine) <br> ([releases](https://github.com/Azure/aks-engine/releases)) | Self-managed clusters on Azure | Azure: incubating <br> API: N/A | [#aks-engine-users](https://kubernetes.slack.com/archives/CU3N85WJK) <br> [GitHub issues](https://github.com/Azure/aks-engine/issues) | [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/) <br> [Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview) |
```
Comment (craiglpeters) [inline README.md]: Now we need to document the progression of each project through the graduation process

File: README.md (hunk 1 above)
Comment (craiglpeters) [reply README.md]: I see the latest release at v1alpha2 still, no?

Comment (CecileRobertMichon) [reply README.md]: Correct, capz latest release is at v1alpha2 and master is currently at v1alpha3.

capi v1alpha3 release just dropped today: https://github.com/kubernetes-sigs/cluster-api/releases/tag/v0.3.0.

Maybe it makes sense to keep this at v1alpha2 until we cut a capz release for v1alpha3 (ETA: a few weeks).

File: README.md [hunk 4]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# Azure Container Compute Upstream Projects
+
+This list of projects is maintained by the Azure container compute upstream team (TODO: how to contact the team). This list is inteded to help you make informed decisions about what projects to use (or not use) in the context of your goals (e.g. proof of concept vs. production). To make this decision you need to consider your goals, your need for formal support, the project's [maturity](#Maturity), governance, version level, and your willingness to work in open source.
```
Comment (craiglpeters) [inline README.md]: Any ideas on the best public way for us to be contacted? Use the issues queue on this project? Create a DL that can get external emails?

Comment (craiglpeters) [reply README.md]: Right now I'm leaning towards GitHub issues, and actually maintaining them properly

Comment (MikkelHegn) [reply README.md]: You'll probably see a lot of issues in this repo, which are related to the actual projects.

Comment (craiglpeters) [reply README.md]: I've tweaked the language in support to mitigate that, and also I can move issues between repos if needed - will help me get visibility to those issues too :)

File: README.md [hunk 5]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# Azure Container Compute Upstream Projects
+
+This list of projects is maintained by the Azure container compute upstream team. This list is inteded to help you make informed decisions about what projects to use (or not use) in the context of your goals (e.g. proof of concept vs. production). To make this decision you need to consider your goals, your need for formal support, the project's [maturity](#Maturity), governance, version level, and your willingness to work in open source.
```
Comment (ritazh) [inline README.md]: intended to

File: README.md (hunk 3 above)
Comment (ritazh) [reply README.md]: For lists like AKS and Azure Stack Hub, maybe use a bullet list as the break is getting lost

File: README.md [hunk 6]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+|   | [Out-of-tree Provider](https://sigs.k8s.io/cloud-provider-azure) <br> ([releases](https://github.com/kubernetes-sigs/cloud-provider-azure/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-cloud-provider-azure) | Use Azure infra for K8s | CNCF: N/A <br> API: alpha | same slack and email <br> [GitHub issues](https://github.com/kubernetes-sigs/cloud-provider-azure/issues) | [Use with AKS Engine](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/docs/cloud-controller-manager.md) |
+|   | [Azure Disk CSI Driver](https://sigs.k8s.io/azuredisk-csi-driver) <br> ([releases](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-azuredisk-csi-driver)  | Enable use of Azure disk volume | CNCF: N/A <br> API: v0.6.0 | same slack and email <br> [GitHub issues](https://github.com/kubernetes-sigs/azuredisk-csi-driver/issues) | [Use with AKS Engine](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/docs/cloud-controller-manager.md) |
```
Comment (ritazh) [inline README.md]: Here the API version for these CSI drivers are the same as the release. Is that intentional? What does API versions mean here? Perhaps we should leave that out. cc @andyzhangx to confirm

File: README.md [hunk 7]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+| Kubernetes Azure Cloud Provider |  |  |  |  |
+|  | [In-tree (legacy)](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/legacy-cloud-providers/azure) <br> ([releases](https://github.com/kubernetes/kubernetes/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-upstream) | Use Azure infra for K8s | CNCF: graduated <br> API: stable | [#provider-azure](https://kubernetes.slack.com/archives/C5HJXTT9Q) <br> [kubernetes-provider-azure@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-provider-azure) <br> [GitHub issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fprovider%2Fazure+) | [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/) |
```
Comment (ritazh) [inline README.md]: Should we also add aks-engine for in-tree provider?

File: README.md [hunk 8]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+|  | [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper) <br> ([releases](https://github.com/open-policy-agent/gatekeeper/releases)) | K8s native Open Policy Agent policy enforcement | CNCF: incubating <br> API: alpha | [#kubernetes-policy](https://openpolicyagent.slack.com/archives/CDTN970AX) <br> [GitHub issues](https://github.com/open-policy-agent/gatekeeper/issues) | [Azure Policy for AKS](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/rego-for-aks) <br> [Azure Policy for AKS Engine](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/aks-engine) |
```
Comment (ritazh) [inline README.md]: API: v1beta1 and v1alpha1 (there are multiple resources)

Comment (ritazh) [reply README.md]: We should also add `Azure Policy for Azure Arc connected clusters`. Once the doc is published, we can add the link later.

File: README.md [hunk 9]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+|  | [Keyvault Flexvol](https://github.com/Azure/kubernetes-keyvault-flexvol) <br> ([releases](https://github.com/Azure/kubernetes-keyvault-flexvol/releases)) | Mount Azure Keyvault secrets into pods | Azure: incubation <br> API: v1 | [GitHub issues](https://github.com/Azure/kubernetes-keyvault-flexvol/issues) | [Use with AKS](https://docs.microsoft.com/en-us/azure/aks/developer-best-practices-pod-security#use-azure-key-vault-with-flexvol) |
```
Comment (ritazh) [inline README.md]: API: NA

File: README.md [hunk 10]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+|  | [Secrets Store CSI Driver](http://sigs.k8s.io/secrets-store-csi-driver) <br> ([releases](https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases)) <br> [Builds](https://testgrid.k8s.io/sig-auth-secrets-store-csi-driver#secrets-store-csi-driver-e2e-vault-postsubmit) | Integrates secrets stores with Kubernetes via a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/) volume | CNCF: ?? <br> API: v1 | [GitHub issues](https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues) |  |
```
Comment (ritazh) [inline README.md]: CNCF: Incubation
API: v1alpha1

File: README.md [hunk 11]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+|  | [Azure KeyVault Provider for Secrets Store CSI Driver](https://github.com/Azure/secrets-store-csi-driver-provider-azure) <br> ([releases](https://github.com/Azure/secrets-store-csi-driver-provider-azure/releases)) | Enables mounting AKV secrets as volumes in K8s pods | Azure: sandbox <br> API: v1 | [GitHub issues](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues) |  |
```
Comment (ritazh) [inline README.md]: API: NA

File: README.md [hunk 12]
```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
...
+| Cloud Native Service Mesh |  |  |  |  |
+|  | [Service Mesh Interface (SMI) Spec](https://smi-spec.io/) | A standard interface for service meshes on Kubernetes | Azure: sandbox <br> API: N/A | [#general](https://smi-spec.slack.com/archives/CJJF5M5NK) <br> [GitHub issues](https://github.com/deislabs/smi-spec/issues) |  |
```
Comment (ritazh) [inline README.md]: cc @michelleN to help provide API versions for SMI resources

File: README.md (hunk 7 above)
Comment (andyzhangx) [reply README.md]: no, in-tree means the code is inside https://github.com/kubernetes/kubernetes, while aks-engine is another external proejct.

File: README.md (hunk 6 above)
Comment (andyzhangx) [reply README.md]: should be `Version` other than `API`

File: README.md (hunk 7 above)
Comment (ritazh) [reply README.md]: @andyzhangx this column means `use in AKS-engine` or `available in AKS-engine`. make sense?

File: README.md (hunk 6 above)
Comment (ritazh) [reply README.md]: ack. so `API: N/A`

File: README.md (hunk 12 above)
Comment (michelleN) [reply README.md]: Should be CNCF Sandbox I think: https://github.com/cncf/toc/pull/336
SMI Spec API versions:
- Traffic Split v1alpha3
- Traffic Target v1alpha1
- Traffic Spec v1alpha2
- Traffic Metrics v1alpha1

File: README.md (hunk 5 above)
Comment (craiglpeters) [reply README.md]: Thanks!

File: README.md (hunk 6 above)
Comment (craiglpeters) [reply README.md]: Ack

Comment (craiglpeters) [reply README.md]: Is there some other pertinent attribute of the project we could put here that would help a user decide on whether it makes sense to use in production?

File: README.md (hunk 7 above)
Comment (craiglpeters) [reply README.md]: I agree - makes sense to indicate that you can use either in-tree or out of tree in aks-engine

File: README.md (hunk 8 above)
Comment (craiglpeters) [reply README.md]: Ack

File: README.md (hunk 9 above)
Comment (craiglpeters) [reply README.md]: Ack

File: README.md (hunk 10 above)
Comment (craiglpeters) [reply README.md]: The CNCF doesn't have a formal graduation process for kubernetes-sigs hosted projects, so the CNCF status is N/A.

File: README.md (hunk 11 above)
Comment (craiglpeters) [reply README.md]: Ack

File: README.md (hunk 12 above)
Comment (craiglpeters) [reply README.md]: The merge is still pending :) I'll include it. Thanks for the API versions

**Pull request merged**